docs/Documentation/Applications/Matlab/batch.md

```diff
@@ -53,6 +53,7 @@ Once the job has finished, the standard output is saved in a file called
 #SBATCH --nodes=1                # Number of nodes
 #SBATCH --job-name=matlabTest              # Name of job
 #SBATCH --account=<account_string>        # Program-based WCID (account string associated with job)
+#SBATCH --partition=standard
```

Comment:
We're still recommending that users don't specify a partition in their job scripts unless using debug. Is there a specific reason for adding the standard partition?

Reply:
I was getting weird behavior during my tests when I did not specify partitions, so I was adding them to be safe. Feel free to remove them.

Reply:
I removed the specification of partition from the batch file and changed module load conda to module load anaconda3.